lib/matplotlib/tests/test_rcparams.py

```diff
@@ -224,16 +228,13 @@ def test_legend_edgecolor():
         yield _legend_rcparam_helper, rc_dict, target, get_func
 
 
-def test_Issue_1713():
+def test_unicode_parsing():
```

Comment:
I think that's more a symptom of Python 2-ness, "Unicode vs str", but it's not strictly correct.